src/utils/obj_ctl.c

```diff
@@ -500,40 +500,38 @@ static struct option ctl_ops[] = {
 };
 
 int
-main(int argc, char **argv)
+shell(int argc, char *argv[])
 {
 	int	rc;
 
-	if (argc < 2) {
-		printf("%s daos|vos [pmem_file]\n", argv[0]);
+	if (argc < 3)
 		goto out_usage;
-	}
 
 	uuid_generate(ctl_ctx.tsc_pool_uuid);
 	uuid_generate(ctl_ctx.tsc_cont_uuid);
 
 	ctl_ctx.tsc_scm_size	= (128 << 20); /* small one should be enough */
-	ctl_ctx.tsc_nvme_size	= (4ULL << 30);
+	ctl_ctx.tsc_nvme_size	= (8ULL << 30);
```

Comment:
Is this change intentional?

Reply:
Yes, it is. Without it, I got this error:

`$ daos shell daos`
ERROR: dmg: pool create failed: server: code = 655 description = "requested NVMe capacity (4.0 GiB) is too small (min 8.0 GiB)"
ERROR: dmg: server: code = 655 resolution = "NVMe capacity should be larger than 8.0 GiB"

I got a similar error on master eac0d15 (10/Oct) while running the original tool:

`$ obj_ctl daos`
ERROR: dmg: pool create failed: server: code = 655 description = "requested NVMe capacity (4.0 GiB) is too small (min 8.0 GiB)"
ERROR: dmg: server: code = 655 resolution = "NVMe capacity should be larger than 8.0 GiB"

Reply:
For the record, this is because of the cluster size defined in https://github.com/daos-stack/daos/blob/master/src/bio/bio_xstream.c#L46

Before, you could request less than 1.0GiB of NVMe per target, but you were still getting at least 1.0GiB. Now the tool lets you know.